package.json

```diff
@@ -1 +1 @@
-{}
\ No newline at end of file
+{}
```

Comment:
I believe we can remove this and `.prettierrc` since we're using Deno to format

Reply:
No